src/pydvl/valuation/methods/_least_core_solving.py

```diff
@@ -0,0 +1,373 @@
+
+    if "max_iters" not in solver_options and solver_options["solver"] == cp.SCS:
+        solver_options["max_iters"] = 10000
+
+    logger.debug("Removing possible duplicate values in lower bound array")
+    b_lb = problem.utility_values
+    A_lb, unique_indices = np.unique(problem.A_lb, return_index=True, axis=0)
+    b_lb = b_lb[unique_indices]
+
+    logger.debug("Building equality constraint")
+    A_eq = np.ones((1, n_obs))
+    # We might have already computed the total utility one or more times.
+    # This is the index of the row(s) in A_lb with all ones.
+    total_utility_indices = np.where(A_lb.sum(axis=1) == n_obs)[0]
+    if len(total_utility_indices) == 0:
+        b_eq = np.array([u(Sample(idx=None, subset=u.training_data.indices))])
+    else:
+        b_eq = b_lb[total_utility_indices]
+        # Remove the row(s) corresponding to the total utility
+        # from the lower bound constraints
+        # because given the equality constraint
+        # it is the same as using the constraint e >= 0
+        # (i.e. setting non_negative_subsidy = True).
+        mask: NDArray[np.bool_] = np.ones_like(b_lb, dtype=bool)
+        mask[total_utility_indices] = False
+        b_lb = b_lb[mask]
+        A_lb = A_lb[mask]
+
+    # Remove the row(s) corresponding to the empty subset
+    # because, given u(∅) = (which is almost always the case,
+    # it is the same as using the constraint e >= 0
+    # (i.e. setting non_negative_subsidy = True).
+    emptyset_utility_indices = np.where(A_lb.sum(axis=1) == 0)[0]
+    if len(emptyset_utility_indices) > 0:
+        mask = np.ones_like(b_lb, dtype=bool)
+        mask[emptyset_utility_indices] = False
+        b_lb = b_lb[mask]
+        A_lb = A_lb[mask]
+
+    _, subsidy = _solve_least_core_linear_program(
+        A_eq=A_eq,
+        b_eq=b_eq,
+        A_lb=A_lb,
+        b_lb=b_lb,
+        non_negative_subsidy=non_negative_subsidy,
+        solver_options=solver_options,
+    )
+
+    values: Optional[NDArray[np.float_]]
+
+    if subsidy is None:
+        logger.debug("No values were found")
+        status = Status.Failed
+        values = np.empty(n_obs)
+        values[:] = np.nan
+        subsidy = np.nan
+    else:
+        values = _solve_egalitarian_least_core_quadratic_program(
+            subsidy,
+            A_eq=A_eq,
+            b_eq=b_eq,
+            A_lb=A_lb,
+            b_lb=b_lb,
+            solver_options=solver_options,
+        )
+
+        if values is None:
+            logger.debug("No values were found")
+            status = Status.Failed
+            values = np.empty(n_obs)
+            values[:] = np.nan
+            subsidy = np.nan
+        else:
+            status = Status.Converged
+
+    return ValuationResult(
+        algorithm=algorithm,
+        status=status,
+        values=values,
+        subsidy=subsidy,
+        stderr=None,
+        data_names=u.training_data.data_names,
+    )
+
+
+@deprecated(
```

Comment:
If I understand correctly, this deprecation notice shouldn't be necessary:
1. Because the deprecation of the config argument is completed in 0.10.0 which is the release which will include the refactor
2. Because lc_solve_problems in this module is not exported (and distinct from the original one in `pydvl.value`